### PR TITLE
completed assessment

### PR DESCRIPTION
**Pull request by namieralsabagh**
# Submitting Week 3 Assessment

### ✅ Completed Files
Mark the following as confirmation that the files in this repository are complete.
- [x ] Coding Challenges
- [ x] Interview Questions
- [ x] React Debugging

### 📝 Feedback
If you would like specific feedback on a particular problem please leave details here:

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -140,6 +138,6 @@ const numbersToAdd3 = []
 
 
    //I have tried thise countless different ways and cannot get it to pass. I have tried looking up solutions to help my thought process and cannot figure this out.  I keep getting the following error message on line 107; > 107 |     expect(accSum(numbersToAdd1)).toEqual([2, 4, 45, 9])
```
Comment (SunkissedQueen) [inline code-challenges.test.js:140]: Look at your expect statement. Have you entered your expected output correctly?

File: react-debugging.html [hunk 2]
```diff
@@ -94,3 +94,4 @@ <h1>Draw a Card</h1>
     </script>
   </body>
 </html>
+
```
Comment (SunkissedQueen) [inline react-debugging.html:97]: Look like you have followed your console. Good work

**Review by SunkissedQueen — COMMENTED** [review]
Your interview questions did not make it to this branch. Let's talk before you merge anything.

**Review by namieralsabagh — COMMENTED** [review]
Thank you for your feedback!

**Review by namieralsabagh — COMMENTED** [review]
Thank you!

**Pull request closed without merging**